Site/ASAB Maestro/Descriptors/mongo.yaml

```diff
@@ -9,26 +9,19 @@ descriptor:
   volumes:
     - "{{SLOW_STORAGE}}/{{INSTANCE_ID}}/data:/data/db"
     - "{{SITE}}/{{INSTANCE_ID}}/conf:/etc/mongo:ro"
-    - "{{SITE}}/{{INSTANCE_ID}}/script:/script:ro"
+    - "{{SITE}}/{{INSTANCE_ID}}/mongo:/script:ro"
 
   command: mongod --config /etc/mongo/mongod.conf
 
 files:
+# This is confusing - I can EITHER specify FOLDER in /Files/... dir OR a write a FILE directly, I was trying to specify unique files from /Files
```

Comment:
I believe that both cases are supported ... and there is a documentation of that somewhere ;-)

Reply:
That's a bold statement

Reply:
http://gitlab.teskalabs.int/asab/asab-remote-control/-/blob/master/remote_control/model/service.py#L390

Reply:
http://gitlab.teskalabs.int/asab/asab-remote-control/-/blob/master/remote_control/model/service.py#L488

Reply:
I see incorrect (old) comment.
Should I test this?